hw/net/rocker/rocker_hw.h

```diff
@@ -146,6 +147,7 @@ typedef struct rocker_tlv {
 enum {
     ROCKER_TLV_CMD_UNSPEC,
     ROCKER_TLV_CMD_TYPE,                /* u16 */
+    ROCKER_TLV_CMD_WORLD,               /* u32 - world id */
```

Comment:
Since the ROCKER_TLV_CMD_TYPE_P4_xxx cmds are already only for P4 world, it seems we don't need this ROCKER_TLV_CMD_WORLD.  In other words, the world is encoded in the cmd type.